rdoc.gemspec

```diff
@@ -46,27 +46,27 @@ RDoc includes the +rdoc+ and +ri+ tools for generating and displaying documentat
     "LEGAL.rdoc",
     "LICENSE.rdoc",
     "README.rdoc",
-    "RI.rdoc",
+    "RI.md",
     "TODO.rdoc",
     "exe/rdoc",
     "exe/ri",
     "lib/rdoc.rb",
-    "lib/rdoc/alias.rb",
-    "lib/rdoc/anon_class.rb",
-    "lib/rdoc/any_method.rb",
-    "lib/rdoc/attr.rb",
-    "lib/rdoc/class_module.rb",
+    "lib/rdoc/code_object/alias.rb",
```

Comment:
Should this list of files be sorted?
To be easy to watch diff, now it is not so.

Reply:
Good point! I’ll open a follow up PR for some cleanups, which will include sorting the list 👍